frontend/containers/account-type-selector/account-type-item/component.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC, useState } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Image from 'next/image';
+
+import { usePress, useFocusWithin } from '@react-aria/interactions';
+import noop from 'lodash-es';
+
+import { AccountTypeItemProps } from './types';
+
+export const AccountTypeItem: FC<AccountTypeItemProps> = ({
```

Comment:
I like the way you implemented this component (from the accessibility point of view). Did you use the Inclusive Components article for the card?

Reply:
I did, yes!  🙂